hw/ip/rv_core_ibex/data/rv_core_ibex.hjson

```diff
@@ -317,6 +317,36 @@
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "CORE.SENSITIVE_DATA.SCA",
```

Comment:
I would use `DATA_REG` or `STATE`. It may make sense to extend the descriptions a bit. I would prefer one of these over `SENSITIVE_DATA` because "sensitive" is somewhat redundant: if the data wasn't sensitive, it probably wouldn't be an asset/need protection in the first place.

Reply:
I created a new `DATA_REG_SW` that I've used in place of `SENSITIVE_DATA` everywhere. I felt expanding the existing `DATA_REG` didn't work too well as it explicitly excluded software accessible registers.